### PR TITLE
add share button

### PR DESCRIPTION
**Pull request by akinwale**

### CODE REVIEW

File: src/page/file/view.js [hunk 1]
```diff
@@ -508,6 +508,20 @@ class FilePage extends React.PureComponent {
     this.setState({ fileViewLogged: true });
   };
 
+  // TODO: Move this to lbry-redux
```
Comment (kauffj) [inline src/page/file/view.js]: Good call, I just wrote a similar function this weekend

File: src/page/file/view.js [hunk 2]
```diff
@@ -508,6 +508,20 @@ class FilePage extends React.PureComponent {
...
+  formatLbryUrlForWeb = url => {
+    return url.replace('lbry://', '/').replace(/#/g, ':');
+  };
+
+  handleSharePress = () => {
+    const { claim, notify } = this.props;
+    if (claim) {
+      const { canonical_url: canonicalUrl, short_url: shortUrl, permanent_url: permanentUrl } = claim;
+      const url = 'https://beta.lbry.tv' + this.formatLbryUrlForWeb(canonicalUrl || (shortUrl || permanentUrl));
```
Comment (kauffj) [inline src/page/file/view.js]: I believe the nested parens are superfluous

Comment (kauffj) [reply src/page/file/view.js]: Also, beta.lbry.tv should be a constant somewhere (it's going to switch to lbry.tv soon)

Comment (akinwale) [reply src/page/file/view.js]: > I believe the nested parens are superfluous

In my defense, prettier did this. I made use of ternary operators.

Comment (kauffj) [reply src/page/file/view.js]: @akinwale that's a pretty good defense!

**Pull request merged**